freqtrade/exchange/exchange.py

```diff
@@ -2246,7 +2246,7 @@ def get_historic_ohlcv(
                 candle_type=candle_type,
             )
         )
-        logger.info(f"Downloaded data for {pair} with length {len(data)}.")
+        logger.info(f"Downloaded data for {pair} from ccxt with length {len(data)}.")
```

Comment:
a user won't care about this.

Reply:
Yes, but leave it as it is, you will always see "with length 0" if  the timerange does not include the latest 2 days. So I add another message and changed this one. Better way may be need to refactor the current code.

Reply:
moved the message to `get_historic_ohlcv` to its single caller
https://github.com/freqtrade/freqtrade/pull/10858/commits/cf0f232635bc14be9e977646f8b0744c25cbc7e0